packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,34 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+
+export enum AccordionHeadingLevel {
+    h1 = "1",
+    h2 = "2",
+    h3 = "3",
+    h4 = "4",
+    h5 = "5",
+    h6 = "6",
+}
+
+export class AccordionItem extends FASTElement {
+    @attr
+    public headinglevel: AccordionHeadingLevel = AccordionHeadingLevel.h2;
+
+    @attr({ mode: "boolean" })
+    public expanded: boolean = false;
+
+    @attr
+    public id: string;
+
+    @observable
```

Comment:
Can we come up with somthing more semantic than "button" - maybe something along the lines of "expandButton", or "expansionControl"?